src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -116,6 +115,7 @@ protected ConsumerConfiguration(ConsumerConfiguration cc) {
         this.memStorage = cc.memStorage;
         this.backoff = cc.backoff == null ? null : new ArrayList<>(cc.backoff);
         this.metadata = cc.metadata == null ? null : new HashMap<>(cc.metadata);
+        this.filterSubjects = cc.filterSubjects == null ? null : new ArrayList<>(cc.filterSubjects);
```

Comment:
This was not released. I released that I need this to be null and not an empty list because it's the only way to determine if the user sets data in the list, which they could set empty, and this is used for comparison against a server version of the config during subscribe to ensure there are no mismatches.

Reply:
I think it's just the nature of the `? :`
Since this.filterSubjects is already null by default I could have just
```
if (cc.filterSubjects != null) {
    this.filterSubjects = new ArrayList<>(cc.filterSubjects);
}
```
If you think that's clearer, I can change it. I would do the same with the other list.
---
The reason I copy it is because I always copy list input to constructors because people have reported bugs because _they_ re-used a list or structure. Probably not necessary, just paranoid.